### PR TITLE
refactor: generate gmud

### PR DESCRIPTION
**Pull request by indigovigente**

### CODE REVIEW

File: dist/licenses.txt [hunk 1]
```diff
@@ -457,6 +457,30 @@ Apache-2.0
    limitations under the License.
 
 
+debug
+MIT
+(The MIT License)
```
Comment (ran-j) [inline dist/licenses.txt:462]: We can have this license here

File: src/controllers/actionController.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import jiraDTO from '../dto/jiraDTO.js';
+
+export default class ActionController{
+
+    constructor(github){
+        this.github = github
+    }
+    async createGMUD(){
+        
+        let validate = new Validate(this.github)
```
Comment (ran-j) [inline src/controllers/actionController.js:15]: just a tip, if you declare a value that won't change use `const` to indicate to the one who is reading that its a constant value

File: src/services/jira.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            await axios.get(url,
+                {
+                    headers: {
+                      Authorization: this.basic_auth,
+                    }
+            
+            }).then((res) => {
+                console.log("Issue válida!")
+                this.verifyJiraIssue = true
+                return true
```
Comment (ran-j) [inline src/services/jira.js]: I thing this retunr will not work here

**Review by ran-j — CHANGES_REQUESTED** [review]
I recommend run a lint in the JS code, and use functions and ifs without `{` is not a good practice.

Check this: https://stackoverflow.com/questions/4797286/are-braces-necessary-in-one-line-statements-in-javascript

**Comment by ran-j** [comment]
Do you think is a good ideia to have the folder [coverage](https://github.com/madeiramadeirabr/action-generate-gmud-sre/tree/refactor/generate-gmud/coverage) and [dist](https://github.com/madeiramadeirabr/action-generate-gmud-sre/tree/refactor/generate-gmud/dist) on the production branch ?

File: src/services/jira.js (hunk 3 above)
Comment (indigovigente) [reply src/services/jira.js]: > Você acha que é uma boa ideia ter a pasta [coverage](https://github.com/madeiramadeirabr/action-generate-gmud-sre/tree/refactor/generate-gmud/coverage) e [dist](https://github.com/madeiramadeirabr/action-generate-gmud-sre/tree/refactor/generate-gmud/dist) no branch de produção?

A cobertura realmente não é necessária, vou fazer a remoção.
A dist sim, porque a própria doc do github actions informa que devemos usar assim

**Pull request merged**